core/src/main/java/org/jruby/ext/pathname/RubyPathname.java

```diff
@@ -406,6 +406,17 @@ public IRubyObject unlink(ThreadContext context) {
         }
     }
 
+    @JRubyMethod(name = "empty?")
+    public IRubyObject empty_p(ThreadContext context) {
+        RubyModule fileTest = context.runtime.getFileTest();
+        if (fileTest.callMethod(context, "directory?", getPath()).isTrue()) {
+            return context.runtime.getDir().callMethod(context, "empty?", getPath());
+        }
+        else {
```

Comment:
Our style is to have '} else {' instead of having closing brackets on their own lines.